docs/stacks-guide-gcp.md

```diff
@@ -132,14 +132,38 @@ GCP Console.
 export PROJECT_ID=[your-demo-project-id]
 export BASE64ENCODED_GCP_PROVIDER_CREDS=$(base64 crossplane-gcp-provider-key.json | tr -d "\n")
 ```
+> Environment variable PROJECT_ID is going to be used in YAML files in the next steps while BASE64ENCODED_GCP_PROVIDER_CREDS
+is only needed for this step.
 
 Now we’ll create our `Secret` that contains the credential and
 `Provider` resource that refers to that secret:
 
 ```
-sed "s/BASE64ENCODED_GCP_PROVIDER_CREDS/$BASE64ENCODED_GCP_PROVIDER_CREDS/g;s/PROJECT_ID/$PROJECT_ID/g" cluster/examples/workloads/kubernetes/wordpress/gcp/provider.yaml | kubectl create -f -
-unset PROJECT_ID
-unset BASE64ENCODED_GCP_PROVIDER_CREDS
+cat > provider.yaml <<EOF
+---
+apiVersion: v1
+data:
+  credentials.json: $BASE64ENCODED_GCP_PROVIDER_CREDS
+kind: Secret
+metadata:
+  namespace: gcp
+  name: gcp-provider-creds
+type: Opaque
+---
+apiVersion: gcp.crossplane.io/v1alpha2
+kind: Provider
+metadata:
+  namespace: gcp
+  name: gcp-provider
+spec:
+  credentialsSecretRef:
+    name: gcp-provider-creds
+    key: credentials.json
+  projectID: $PROJECT_ID
+EOF
+
+kubectl apply -f provider.yaml
+# Example YAML exists in https://github.com/crossplaneio/crossplane/blob/master/cluster/examples/workloads/kubernetes/wordpress/gcp/provider.yaml as well.
```

Comment:
Nitpick: because urls are so long, I usually like to put them at the end of a sentence, in case people stop reading when they get to the url. For example, I would normally say something more like `# Example YAML also exists in URLHERE`.